lib/va_profile/prefill/military_information.rb

```diff
@@ -0,0 +1,372 @@
+      # by a call to Lighthouse BRD.
+      COMBINED_SERVICE_BRANCHES = [
+        'Army Air Corps or Army Air Force',
+        'Air Force Academy',
+        'Air Force',
+        'Air Force Reserve',
+        'Air Force Reserves',
+        'Air National Guard',
+        'Army Reserve',
+        'Army Reserves',
+        'Army',
+        'Army National Guard',
+        'Coast Guard Academy',
+        'Coast Guard',
+        'Coast Guard Reserve',
+        'Coast Guard Reserves',
+        'Marine Corps',
+        'Marine Corps Reserve',
+        'Marine Corps Reserves',
+        'Merchant Marine',
+        'Naval Academy',
+        'Navy',
+        'National Oceanic & Atmospheric Administration',
+        'NOAA',
+        'Navy Reserve',
+        'Navy Reserves',
+        'Other',
+        'Public Health Service',
+        'Space Force',
+        'US Military Academy',
+        "Women's Army Corps"
+      ].freeze
+
+      attr_reader :disability_service, :military_personnel_service
+
+      def initialize(user)
+        @disability_service = VAProfile::Disability::Service.new(user)
+        @military_personnel_service = VAProfile::MilitaryPersonnel::Service.new(user)
+      end
+
+      # @return [Boolean] true if veteran is paid for a disability
+      #  with a high disability percentage
+      #
+      # Rubocop wants this method to be named va_service_connected? but is_va_service_connected
+      # is the name of the method we're replacing.
+      #
+      # rubocop:disable Naming/PredicateName
+      def is_va_service_connected
+        combined_service_connected_rating_percentage >= HIGHER_DISABILITY_RATING
+      end
+      # rubocop:enable Naming/PredicateName
+
+      # @return [Boolean] true if veteran is paid for a disability
+      #  with a low disability percentage
+      def compensable_va_service_connected
+        LOWER_DISABILITY_RATINGS.include?(combined_service_connected_rating_percentage)
+      end
+
+      # @return [String] If veteran is paid for a disability, this method will
+      #  return which type of disability it is: highDisability or lowDisability
+      def va_compensation_type
+        high_disability = is_va_service_connected
+        low_disability = compensable_va_service_connected
+
+        if high_disability
+          'highDisability'
+        elsif low_disability
+          'lowDisability'
+        end
+      end
+
+      # @return [Boolean] true if the user is currently
+      #  serving in active duty
+      def currently_active_duty
+        currently_active_duty_hash[:yes]
+      end
+
+      # @return [Hash] currently active duty data in hash format
+      def currently_active_duty_hash
+        is_active = false
+
+        service_episodes_by_date.each do |episode|
+          if episode.end_date && (episode.end_date.empty? || DateTime.parse(episode.end_date).to_date.future?)
+            is_active = true
+            break
+          end
+        end
+
+        { yes: is_active }
```

Comment:
This hash is quite silly, do we really have to propagate it?

Reply:
I agree 😂. There are a number of forms that expect this format, so I'd rather not change it up.